src/app/client/src/app/modules/core/services/copy-content/copy-content.service.ts

```diff
@@ -80,12 +75,11 @@ export class CopyContentService {
           })
         );
       }));
-    }
   }
   /**
    * @since - 1.#SH-66 || 2.#SH-362
    * @param  {ContentData} contentData
-   * @description - API to copy a textbook as a course.
+   * @description - API to copy a textbook as a course & question set.
```

Comment:
please do not change a method which says copy as course
create a new method and call that